src/aat/resources/features/F-042/S-245.td.json

```diff
@@ -0,0 +1,31 @@
+{
+  "_guid_": "S-245",
+  "_extends_": "F-042_Test_Data_Base",
+  "title": "must return 403 when request does not provide valid authentication credentials",
+  "specs": [
+    "an active profile in CCD",
+    "does not provide valid authorization credentials for an operation in CCD",
+    "has a 403 Forbidden code"
+  ],
+  "request": {
+    "headers": {
+      "Authorization": "Invalid_Authorization"
+    }
+  },
+  "expectedResponse": {
+    "_extends_": "Common_403_Response",
+    "headers": {
+      "Cache-Control" : "no-cache, no-store, max-age=0, must-revalidate",
+      "Date" : "[[ANY]]",
+      "Expires" : "0",
```

Comment:
Cache-Control, Date and Expires are included in Common_Response_Headers, can be removed from here

Reply:
Removed the headers.